.travis.yml

```diff
@@ -30,7 +42,7 @@ install:
   - cd $TRAVIS_BUILD_DIR
   # Create unix makefiles
   - cp $HOME/deps/travisci/cmake/FindFreetype.cmake CMake/FindFreetype.cmake
-  - cmake -DCMAKE_CXX_COMPILER="g++-5" -DLUA_PROGRAM_PATH=`which lua` -DWITH_AUDIO=ON -DWITH_MOVIES=ON -DWITH_LUAJIT=OFF -DCMAKE_INCLUDE_PATH="$CMAKE_INCLUDE_PATH;$CMAKE_INCLUDE_PATH/SDL2" -DCMAKE_LIBRARY_PATH="$CMAKE_LIBRARY_PATH" -DLUA_LIBRARY="$CMAKE_LIBRARY_PATH/liblua.so" -Bfresh -H. --debug-output
+  - cmake -DCMAKE_CXX_COMPILER="g++-5" -Bfresh -H. --debug-output -DWITH_AUDIO=ON -DWITH_MOVIES=ON -DWITH_LUAJIT=OFF -DCMAKE_INCLUDE_PATH="$CMAKE_INCLUDE_PATH;$CMAKE_INCLUDE_PATH/SDL2" -DCMAKE_LIBRARY_PATH="$CMAKE_LIBRARY_PATH" -DLUA_PROGRAM_PATH="$HOME/lua-5.3.2/install/bin/lua" -DLUA_LIBRARY="$HOME/lua-5.3.2/install/lib/liblua.a" -DLUA_INCLUDE_DIR="$HOME/lua-5.3.2/include/lua5.3/"
```

Comment:
Since you have compiled lua 5.3.2 for Ubuntu, perhaps you could replace the one in CorsixTH/deps. I believe that repo still has 5.3.1